.gitignore

```diff
@@ -1,3 +1,4 @@
 .build/
 vendor
 composer.lock
+/.phpunit.result.cache
```

Comment:
Please point `phpunit/phpunit`s result cache into a location in the already ignored `.build` directory.

If necessary, we can create the directory before running tests.